appinfo/routes.php

```diff
@@ -25,7 +25,7 @@
 		//Retrieve Metadata of a list of files
 		[
 			'name' => 'metadata#get_metadata',
-			'url'  => '/metadata/{fileList}',
+			'url'  => '/metadata?fileList={fileList}',
```

Comment:
@oparoz I made it to the above route. But still the same error is being displayed.